app/forms/decidim/collaborations/admin/collaboration_form.rb

```diff
@@ -34,8 +34,8 @@ class CollaborationForm < Decidim::Form
                   numericality: { only_integer: true, greater_than: 0 }
 
         validates :target_amount,
-                  numericality: { only_integer: true, greater_than: 0 },
-                  allow_blank: true
+                  presence: true,
```

Comment:
mm, I remember touched this before. Target amount is not required for every campaigns.

Reply:
Ah ok, then it's the test that was wrong, I'll fix the test!

Reply:
Yes, that's my fault, I've fixed it and not updated the test

Reply:
Fixed!